impeller/entity/geometry/circle_tessellator.h

```diff
@@ -0,0 +1,136 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#pragma once
+
+#include <functional>
+#include <vector>
+
+#include "flutter/impeller/geometry/matrix.h"
+#include "flutter/impeller/geometry/point.h"
+#include "flutter/impeller/geometry/scalar.h"
+
+namespace impeller {
+
+/// @brief  A structure to store the sine and cosine of an angle.
+struct Trig {
+  /// Construct a Trig object from a given angle in radians.
+  Trig(Radians r) : cos(std::cos(r.radians)), sin(std::sin(r.radians)) {}
```

Comment:
Add `// NOLINTBEGIN(google-explicit-constructor)` and `// NOLINTEND(google-explicit-constructor)` as to not add more noise for @matanlurey 's clang tidy work

Reply:
I'll just make it explicit...